app/code/Magento/InventorySalesAdminUi/Model/GetStockSourceLinksBySourceCodeInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySalesAdminUi\Model;
+
+use Magento\InventoryApi\Api\Data\StockSourceLinkSearchResultsInterface;
+
+/**
+ * Sugar service for find StockSourceLinks by source code
+ *
+ * @api
```

Comment:
taking our current strategy of modules decomposition.
API, Implementation, AdminUi -it's not corret to intorduce `@api` interfaces into the AdminUi modules.
Because of Implementation and UI modules should collaborate to each other via services located in API modules only